mhcnames/species.py

```diff
@@ -81,21 +81,21 @@
         prefix_to_species_name[prefix] = species
 
 
-def split_species_prefix(name):
+def split_species_prefix(name, seps="-:_ "):
```

Comment:
Not clear why `seps="-:_ "` vs. just `seps="-"`; seems like the former could result in unexpected behavior?